azure-pipelines.yml

```diff
@@ -1,245 +1,247 @@
-- template: build/stage-uitests-ios.yml
-- template: build/stage-uitests-android.yml
+variables:
+  IsLightBuild: $[eq(variables['Build.Reason'], 'PullRequest')]
+  IsCanaryBranch: $[startsWith(variables['Build.SourceBranch'], 'refs/heads/canaries/')]
+
+jobs:
+- job: Windows_Android
+  timeoutInMinutes: 90
+
+  strategy:
+    maxParallel: 3
+    matrix:
+      Android_Store:
+        ApplicationPlatform: AnyCPU
+        ArtifactName: Android
+        BuildForPlayStore: True
+        VersionCodeOffset: 1
+      Android_AppCenter:
+        ApplicationPlatform: AnyCPU
+        ArtifactName: Android
+        BuildForPlayStore: False
+        VersionCodeOffset: 1
+
+  pool:
+    vmImage: windows-2022
+
+  variables:
+  - group: 'UADO Keystore' # Import all variables for the signing from the library in azure devops
+
+  steps:
+  - template: build/stage-build-android-xamarin.yml
+
+- job: Windows_Android_Mobile
+  strategy:
+    maxParallel: 3
+    matrix:
+      Android_Store:
+        ApplicationPlatform: AnyCPU
+        ArtifactName: Android
+        BuildForPlayStore: True
+        VersionCodeOffset: 1
+
+  pool:
+    vmImage: windows-2022
+
+  variables:
+  - group: 'UADO Keystore' # Import all variables for the signing from the library in azure devops
+
+  steps:
+  - template: build/stage-build-android-mobile.yml
+
+- job: Windows_UWP
+  pool:
+    vmImage: windows-2019
+
+  variables:
+  - group: 'UADO Keystore' # Import all variables for the signing from the library in azure devops
+  - name: ApplicationPlatform
+    value: x64
+  - name: ArtifactName
+    value: UWP
+  - name: VersionCodeOffset
+    value: 50
+
+  steps:
+  - template: build/stage-build-uwp.yml
+
+- job: Windows_SkiaGtk
+  pool:
+    vmImage: windows-2019
+
+  variables:
+  - group: 'UADO Keystore' # Import all variables for the signing from the library in azure devops
+
+  steps:
+  - template: build/stage-build-skia-gtk.yml
+
+- job: macOS_Xamarin
+  pool:
+    vmImage: macos-12
+
+  variables:
+  - name: SkipUnknownFrameworks
+    value: true # Used by TargetFrameworks.Filtering package
+  - group: unoplatform.apple.ios.appstore.distribution
+  - name: ApplicationPlatform
+    value: iPhoneSimulator
+  - name: ArtifactName
+    value: macOS
+  - name: VersionCodeOffset
+    value: 50
+
+  steps:
+  - task: InstallAppleCertificate@2
+    displayName: Install Certificate
+    inputs:
+      certSecureFile: UnoPlatform-Apple-Distribution.p12
+      certPwd: $(appleappstorecertificatepassword)
+      keychain: temp
+      # disabled because of azure devops failing to uninstall as of 2022-11-16
+      # deleteCert: true
+)
+
+  - task: InstallAppleCertificate@2
+    displayName: Install Certificate
+    inputs:
+      certSecureFile: UnoPlatform-MacInstaller.p12
+      certPwd: $(appleappstorecertificatepassword)
+      keychain: temp
+      # disabled because of azure devops failing to uninstall as of 2022-11-16
+      # deleteCert: true
+)

```

Comment:
This looks like it's not required :) This is the reason for the build failing.

Reply:
@ajpinedam